moduleroot/spec/spec_helper.rb.erb

```diff
@@ -36,6 +36,9 @@ RSpec.configure do |c|
   <%- if @configs['hiera_config'] -%>
   c.hiera_config = '<%= @configs['hiera_config'] %>'
   <%- end -%>
+  <%- if @configs['hiera_execute_config'] -%>
```

Comment:
should this be an elsif ? is it valid to have both hiera_config and hiera_config ?

Reply:
@tphoney make sense. Will make the changes and also update the documentation.

Reply:
@tphoney Thank you. Incorporated the review comments and tested the changes.